conda-forge.yml

```diff
@@ -1,2 +1,13 @@
+min_py_ver: '37'
```

Comment:
Please, do not touch these files, I mean, ``.travis.yml`` and ``conda-forge.yml``

Reply:
Okay. Perhaps I missed something but the documentation [here](https://conda-forge.org/docs/maintainer/conda_forge_yml.html) did not make this clear.